src/components/Inputs/input.module.scss

```diff
@@ -1438,6 +1447,93 @@
     &.input-stretch {
         width: 100%;
     }
+
+    &.isExpandable {
+        .expandable-wrapper {
+            position: relative;
+        }
+
+        .input-group {
+            position: absolute;
+            left: 0;
+            z-index: 10;
+
+            input {
+                width: 0;
+            }
+
+            .clear-icon-button {
+                right: -12%;
+            }
+        }
+
+        .expandable-thumb {
+            position: absolute;
+            left: -3px;
+        }
+
+        .overlay {
+            position: absolute;
+            left: 0;
+            top: 0;
+            height: 100%;
+            width: 200%;
+            z-index: -1;
+            background: $expandable-header-gradient;
+        }
+
+        input,
+        .input-group,
+        .overlay {
+            transition: all $motion-duration-fast $motion-easing-easeout;
+        }
+
+        // Set Rules
+        .input-group:hover > input,
+        input:focus,
+        input:not(:placeholder-shown) {
+            @each $size-name, $size-value in $input-size-map {
+                &.input-#{$size-name} {
+                    width: auto;
+                }
+            }
+
+            ~ .actionWrapper {
+                display: block;
+            }
+
+            ~ .expandable-thumb {
+                display: none;
+            }
+        }
+
+        // Unset Rules
+        @each $size-name, $size-value in $input-size-map {
+            &.input-#{$size-name} > .expandable-wrapper {
+                min-width: $size-value;
+                min-height: $size-value;
+            }
+
+            .input-group:not(:hover)
```

Comment:
seemingly focus or focus-within might also be relevant here?
strange selector wrapping and indentation

Reply:
some last moment change. will fix this.

Reply:
We're setting the input style when the input-group has not hovered.

Reply:
I understand the selector, but is this sufficient? When a user tabs into the input, things don't look as they should. Not sure if this is directly related, but needs to be addressed.